quickstart/deploy-mayastor.md

```diff
@@ -75,15 +75,26 @@ kubectl -n mayastor get pods --selector=app=nats
 
 {% tab title="Example Output" %}
 ```text
-NAME          READY   STATUS    RESTARTS   AGE
-nats-0        3/3     Running   0          61s
+NAME     READY   STATUS    RESTARTS   AGE
+nats-0   2/2     Running   0          50s
+nats-1   2/2     Running   0          30s
+nats-2   1/2     Running   0          10s
 ```
 {% endtab %}
 {% endtabs %}
 
 ### etcd
 
-Mayastor uses [etcd](https://etcd.io/), a distributed, reliable key-value store, to persist configuration.  The steps described below deploy a dedicated, clustered etcd  instance for Mayastor's own use.  This is the only configuration supported by this release.  
+Mayastor uses [etcd](https://etcd.io/), a distributed, reliable key-value store, to persist configuration.  The steps described below deploy a dedicated, clustered etcd  instance for Mayastor's own use. This is the only configuration supported by this release.  
+
+To deploy the PersistentVolumes that will be used by the etcd in the next step, execute:
+{% tabs %}
+{% tab title="Command \(GitHub Latest\)" %}
+```text
+kubectl apply -f https://raw.githubusercontent.com/openebs/mayastor/develop/deploy/etcd/storage/localpv.yaml
```

Comment:
This file has a storage-class named "manual", which needs to be defined. Also, it's better to use K8s local PV hostpath instead of regular hostpath for the benefits that K8s local PV hostpath brings over the traditional hostpath.